src/CMakeLists.txt

```diff
@@ -80,20 +80,20 @@ endif (NOT EIGEN_TAR_SOURCE)
 # specify an alternative path to the PyXB tarball:
 # -DPYXB_TAR_SOURCE=/path/to/PyXB-x.x.x.tar.gz
 
-set(PYXB_VERSION "1.2.6")
-set(PYXB_TAR_MD5 4303573fc7094ce4664c5b71cd4bdb48)
+# set(PYXB_VERSION "1.2.6")
```

Comment:
do we want to keep this commented out or remove it altogether.

Reply:
we may need to update the [conflunce](https://cwiki.apache.org/confluence/display/MADLIB/Third+Party+Components) page too.
Also, will this require any changes to the license file?